MAUI/Accordion/getting-started.md

```diff
@@ -358,10 +352,10 @@ private void SfAccordion_Collapsing(object sender, Syncfusion.Maui.Accordion.Exp
 
 ### Collapsed Event 
 
-The `Collapsed` event is triggered when the accordion item is collapsed. You can execute your own code when this event occurs.
+The [Collapsed](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.Accordion.SfAccordion.html#Syncfusion_XForms_Accordion_SfAccordion_Collapsed) event is triggered when the accordion item is collapsed. You can execute your own code when this event occurs.

```

Comment:
Provided links are Xamarin APIs. Please update with MAUI link.

Reply:
Addressed the concern provided